data/basics/teal.go

```diff
@@ -28,11 +28,11 @@ import (
 type DeltaAction uint64
 
 const (
-	// SetBytesAction indicates that a TEAL byte slice should be stored at a key
-	SetBytesAction DeltaAction = 1
-
 	// SetUintAction indicates that a Uint should be stored at a key
-	SetUintAction DeltaAction = 2
+	SetUintAction DeltaAction = 1
+
+	// SetBytesAction indicates that a TEAL byte slice should be stored at a key
+	SetBytesAction DeltaAction = 2
```

Comment:
@justicz Why are these switched here?

Reply:
Unintentionally -- these changed on master at some point and I must have accidentally copied this from an older version.